plugins/modules/fusion_se.py

```diff
@@ -283,30 +410,63 @@ def main():
                 module.params["availability_zone"]
             )
         )
-    if module.params["network_interface_groups"] and not get_nifg(module, fusion):
-        module.fail_json(
-            msg="Not all of the network interface groups exist in the specified AZ"
-        )
-    if module.params["addresses"]:
-        for address in module.params["addresses"]:
-            if not is_valid_network(address):
+
+    if (
+        "addresses" in module.params
+        or "gateway" in module.params
+        or "network_interface_groups" in module.params
+    ):
```

Comment:
If someone is still using the deprecated parameter,  we should show a warning message "<param_name> is deprecated and will be removed in the version 2.0"

Reply:
use `module.warn` for this

Reply:
done